pkg/operator/encryption/deployer/staticpod.go

```diff
@@ -123,65 +120,51 @@ const revisionLabel = "revision"
 // Once a converged revision has been determined, it can be used to determine
 // what encryption config state has been successfully observed by the API servers.
 // It assumes that podClient is doing live lookups against the cluster state.
-func getAPIServerRevisionOfAllInstances(nodes []operatorv1.NodeStatus, podClient corev1client.PodInterface) (string, error) {
+func getAPIServerRevisionOfAllInstances(revisionLabel string, nodes []operatorv1.NodeStatus, podClient corev1client.PodInterface) (string, error) {
```

Comment:
nit: this arg `revisionLabel`could be internal to this function `getAPIServerRevisionOfAllInstances`, no need to expose it.

Reply:
it's intentional. A static-pod deployer will use  a different label.